src/components/RegisterLanding.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import NoOnboarding from '../pages/NoOnboarding';
+import StyledFieldset from './StyledFieldset';
+
+export default function RegisterLanding({ setRegisterStep }) {
+  const [isRegistered, setIsRegistered] = useState(null);
+
+  const handleRadioChoice = (e) => {
+    const boolValue = e.target.value === 'yes';
+    setIsRegistered(boolValue);
+  };
+
+  return (
+    <>
+      {/* {console.log("isRegistered:",isRegistered)} */}
+      {isRegistered === 'not' ? (
```

Comment:
Instead of checking against a string, which is something that can be easily typoed, let's use a boolean instead.
So `isRegistered` can either equal `true` or `false`